test/minitest_helper.rb

```diff
@@ -66,9 +66,10 @@ def refute_raises *exp
 
 ENV['RACK_ENV'] = 'test'
 
- # comment this out to send traces to the collector configured in `env`
- # make sure to set APPOPTICS_SERVICE_KEY
 ENV['APPOPTICS_GEM_TEST'] = 'true'
+ENV['APPOPTICS_REPORTER'] = 'file'
+ENV['APPOPTICS_COLLECTOR'] = '/tmp/appoptics_traces.bson'.freeze
```

Comment:
what is the point of freezing a string? `ENV['APPOPTICS_COLLECTOR']` can still be reassigned, right? why not `.freeze` all the strings? (i guess all string literals are frozen starting in ruby 3 - https://www.honeybadger.io/blog/when-to-use-freeze-and-frozen-in-ruby/)

Reply:
I tried to optimize things by freezing strings, but it is a half hearted attempt and I'm glad ruby is gradually taking it of my plate.